src/cargo/ops/fix.rs

```diff
@@ -132,6 +128,7 @@ pub fn fix(ws: &Workspace<'_>, opts: &mut FixOptions<'_>) -> CargoResult<()> {
 
     let rustc = opts.compile_opts.config.load_global_rustc(Some(ws))?;
     wrapper.arg(&rustc.path);
+    wrapper.args(&opts.primary_unit_args);
```

Comment:
pass via wrapper's argument list. `FixArgs::get()` already grabs all but a few special args and forwards them to the subprocess. This change just exposes that feature to the end user.